### PR TITLE
Submit bpmn-js@17.6.4

### LINKED ISSUE #238: Labels of Pools and Lanes on the left instead of top

### LINKED ISSUE #239: Additional vertical line in each Pool

### PR DESCRIPTION
**Pull request by barmac**
Closes #239
Closes #238

### CODE REVIEW

**Comment by barmac** [comment]
Note that we can now render vertical pools but we still cannot model them in the tool without plugins.

**Comment by falko** [comment]
@tstephen could you have a look at the Maven build?

File: bpmn.io (Camunda Modeler) 17.6.4/A.1.0-export.bpmn [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<definitions xmlns="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:omgdi="http://www.omg.org/spec/DD/20100524/DI" xmlns:omgdc="http://www.omg.org/spec/DD/20100524/DC" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="sid-38422fae-e03e-43a3-bef4-bd33b32041b2" targetNamespace="http://bpmn.io/bpmn" exporter="Camunda Modeler" exporterVersion="5.0.0">
```
[the remark this answers is not in the document]
Comment (barmac) [reply bpmn.io (Camunda Modeler) 17.6.4/A.1.0-export.bpmn:2]: I'll update this.

[the remark this answers is not in the document]
Comment (barmac) [reply bpmn.io (Camunda Modeler) 17.6.4/A.1.0-export.bpmn]: Fixed.

**Review by tstephen — APPROVED** [review]

**Pull request merged**